<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AForm</class>
 <widget class="QWidget" name="AForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>389</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add a task</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="taskLabel">
     <property name="text">
      <string>Task description</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="descLineEdit"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="timeLabel">
     <property name="text">
      <string>Time of the task</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QTimeEdit" name="timeEdit"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="placeLabel">
     <property name="text">
      <string>The place of the task</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLineEdit" name="placeLineEdit"/>
   </item>
   <item row="6" column="0">
    <widget class="QPushButton" name="addButton">
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>83</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="8" column="0">
    <widget class="QPushButton" name="txtImportButton">
     <property name="text">
      <string>Import from a text file</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QPushButton" name="csvImportButton">
     <property name="text">
      <string>Import from a CSV file</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QPushButton" name="binImportButton">
     <property name="text">
      <string>Import from a binary file</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
